### PR TITLE
add Dan Adam's blog post on phishing

### PR DESCRIPTION
**Pull request by WheresAlice**
I've set the date for this for Monday, but I'm out of the country and mostly offline from Saturday 27th for a week so won't be able to respond to any requests for edits.

### CODE REVIEW

**Comment by medains** [comment]
Would be nice to provide an analysis of how typical customers are protected, since 3-4 of these items do not protect the majority of customers; and many of the others depend on their email provider doing the "right thing" - but I can't fault the post as is.

**Comment by sbg-danadams** [comment]
What's our typical audience for the blog? The article was more aimed at those working in a corporate environment, following the pattern of other recent blog posts ("kafka cluster sizing", "Using Chef Search", "Kanban's Geological Record", "Increasing Login Capacit by Using Node.js") rather than being targeted at typical home user. I also wanted to keep the measures approximately in "order"/sequence that they fire/following the progress of the phishing attack through and it would disrupt the flow to split this I think @medains. I guess we could flag them with tags like "[corporate users only]" after each numbered point if you thought that would add something?

**Comment by medains** [comment]
Looking at the analytics, our typical audience is readers coming from HN to read Tom's post... :)
I think it would be worth putting a tag on the post to differentiate it in random web searches from more general "protect yourself from phishing" articles.

File: _posts/2016-02-29-phishing.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+tags:       security
+---
+
+Preventing phishing attacks - an example of defense in depth
+
+"Phishing" attacks are attempts to acquire information such as usernames, passwords, and credit card details by masquerading as a trustworthy entity. The name derives from the similarity of using a bait in an attempt to catch a victim from a pool and the "bait" normally consist of an electronic message such as an email sent to a large group with the hope that some small percentage of targeted victims will fall for the ploy.
+
+A phishing email is generally sent by a malicious source but purporting to be from a known or trusted source such as a company's IT department or a popular social media website or high street bank. Phishing emails will often either have a malicious payload such as a word document with a macro, or else contain links to websites that are either infected with malware or attempt to trick people into entering their credentials.
+
+Typically the email itself, or the website linked to from the email use corporate imagery, branding and fonts to attempt to provide a look and feel are almost identical to the genuine, legitimate business being impersonated.
```
Comment (anatomic) [inline _posts/2016-02-29-phishing.md]: I think this needs a bit of reworking, maybe something like:

> "Typically the email itself (or the website linked to from the email) use corporate imagery, branding and fonts to provide an experience which is almost identical to the genuine business being impersonated"

File: _posts/2016-02-29-phishing.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+Phishing attacks are difficult to entirely prevent against, but are a good example of a situation in which "defense in depth" is appropriate - no one measure on its own is effective, but by implementing a range of anti-phishing countermeasures, the effectiveness of phishing attacks can be greatly reduced. So what can be done?
+
+### 1. Server-side Email filtering
+
+The first line of defense needed is anti-spam and email filtering for inbound email, to prevent emails from being received by your employees in the first place. Various technologies can be used including anti-spam filter based on BLs and IP reputation services, a server-side antivirus scan for attachments, and MX record/domain verification using Sender Policy Framework (SPF). Emails can be rejected by your mailserver so that they never make it to your users.
```
Comment (anatomic) [inline _posts/2016-02-29-phishing.md]: Either "an anti-spam filter..." or "including anti-spam filters...".

Also, maybe "Emails can then be rejected by..."

File: _posts/2016-02-29-phishing.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+### 2. User training and internal policies
+
+Inevitably, some emails are going to get through your email filters, no matter how infrequently. It is important to reduce the likelihood that employees are tricked into falling for the phishing scams by providing training as to how to recognise likely phishing attempts. It helps if advice is consistent - if users are aware that internal IT staff will never ask them for their passwords, then they are less likely to fall for a phishing attack asking them for their password. There are also excellent online tools such as [VirusTotal](https://www.virustotal.com/) which allow users to upload suspected attachments before opening them in order to check whether they contain any viruses.
+
+### 3. Mail client software
+
+Mail client software has a part to play in providing warning to users that opening attachments from emails or clicking on links may provide some element of risk of virus and phishing attacks and provide reminders to users of their training at the time that the phishing email is received.
```
Comment (anatomic) [inline _posts/2016-02-29-phishing.md]: Could this be broken up into smaller sentences? It's a bit long and awkward at the moment

File: _posts/2016-02-29-phishing.md (hunk 1 above)
Comment (sbg-danadams) [inline _posts/2016-02-29-phishing.md]: How do we actually edit this?

File: _posts/2016-02-29-phishing.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+### 5. Internet Content Filtering (ICF) and restricted outbound internet access.
+
+A further measure that can help is internet content filtering on outbound web requests from employee workstations and laptops. ICF is a mechanism that intercepts outbound web requests and verifies whether they are to be permitted. Very often a phishing email will not have a malicious payload itself, in order to improve its chances of being delivered to targeted victims. It will instead contain a seemingly innocuous word document or Excel document attachment with a simple macro that executes on file open and attempts to download the real virus payload, or just a link to a website. Usually this file or webpage will be hosted on known malicious domains and ICF can actively block access to these domains so that the payload cannot be downloaded or form that attempts to capture their password cannot be loaded.
```
Comment (anatomic) [inline _posts/2016-02-29-phishing.md]: Last sentence just needs a bit of finessing - maybe something like "Usually this file or webpage will be hosted on a domain known to be malicious. ICF can be used to actively block access to such domains, preventing the payload from being downloaded or the form designed to capture passwords from loading in the user's browser."

File: _posts/2016-02-29-phishing.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+### 8. Password management, expiry
+
+Finally, if the worst does happen, and a phishing email is received by a user and successfully tricks them into entering their password, all is not lost. By enforcing strong password management, you can ensure that users are using different passwords for each site or service they use, meaning that the damage in the credential loss is known and limited. By enforcing frequent password rotation/expiry you can also ensure that the window during which a malicious party can exploit any captured credentials is limited.
+
+The above example is not an exhaustive list of every measure that can be taken against phishing attacks, but does demonstrate that very often the strength of a security solution lies not at any single layer or component, but in a so-called defense in depth approach, in which a range of measures at different layers and types are stacked to provide a robust overarching protection against a given threat
```
Comment (anatomic) [inline _posts/2016-02-29-phishing.md]: Maybe title this summing up paragraph "Defense in Depth" or similar so it is separated out from point number 8.

**Comment by anatomic** [comment]
Going to close this one as we've updated and re-requested from another source (as Alice is out-of-office today)

**Pull request closed without merging**